tests/e2e/kcs/test_selinux_relabel_solution.py

```diff
@@ -0,0 +1,322 @@
+import logging
+import pytest
+import yaml
+import random
+import time
+import tempfile
+from datetime import datetime
+
+from ocs_ci.framework import config
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.ocs.exceptions import PodNotCreated, CommandFailed
+from ocs_ci.ocs.resources import pod as res_pod
+from ocs_ci.ocs.resources.pod import wait_for_pods_to_be_running
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.utility.templating import dump_data_to_temp_yaml
+
+log = logging.getLogger(__name__)
+
+
+class TestSelinuxrelabel(E2ETest):
+    def create_deploymentconfig_pod(self, **kwargs):
+        """
+        Create deployment pod.
+
+        Returns:
+            object: helpers.create_pod instance
+
+        """
+        try:
+            pod_obj = helpers.create_pod(
+                interface_type=constants.CEPHFS_INTERFACE,
+                pvc_name=self.pvc_obj.name,
+                namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+                sa_name=self.service_account_obj.name,
+                dc_deployment=True,
+                pod_dict_path=constants.PERF_DC_YAML,
+                **kwargs,
+            )
+        except Exception as e:
+            log.exception(
+                f"Pod attached to PVC {pod_obj.name} was not created, exception [{str(e)}]"
+            )
+            raise PodNotCreated("Pod attached to PVC was not created.")
+        return pod_obj
+
+    def get_app_pod(self):
+        """
+        Get cephfs app pod
+
+        Returns:
+            object: app pod instance
+
+        """
+        pod_obj = res_pod.get_all_pods(
+            namespace=config.ENV_DATA["cluster_namespace"],
+            selector=[self.pod_selector],
+            selector_label=constants.DEPLOYMENTCONFIG,
+        )
+        pod_name = self.pod_selector + "-1-deploy"
+        for pod_obj in pod_obj:
```

Comment:
you are using the same var name which can create conflicts, better to use a diff var name